src/org/labkey/test/tests/MessagesLongTest.java

```diff
@@ -617,7 +663,10 @@ private void createNewMessage(String title, String body)
     private void basicMessageTests()
     {
         log("Add search to project");
+        SiteNavBar siteNavBar = new SiteNavBar(getDriver());
+        siteNavBar.enterPageAdminMode();
         _portalHelper.addWebPart("Search");
+        siteNavBar.exitPageAdminMode();
```

Comment:
Manually entering page admin mode shouldn't be necessary here. `PortalHelper.addWebPart` handles that.